### PR TITLE
Create james_vincent_mead.md

### PR DESCRIPTION
**Pull request by jvmead**

### CODE REVIEW

**Comment by jvmead** [comment]
@klieret I had already gone down this route - does everything check out?

File: _profiles/james_vincent_mead.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage:                                   # your personal homepage (full url)
+twitter: jvmead                             # your twitter ID (not full URL, no leading '@')
+gitlab:                                     # your gitlab ID (not full URL)
+bitbucket:                                  # your bitbucket ID (not full URL)
+orcid: 0000-0003-0875-2533                  # your orcid ID (not full URL)
+linkedin: james-mead                        # your linkedin ID (not full url, i.e. the last bit of the url to your profile)
+email: jmead@nikhef.nl
+
+# Training WG
+training_roles: []              # subset of [instructor, mentor, facilitator, author], can stay empty ([])
```
Comment (klieret) [inline _profiles/james_vincent_mead.md:18]:
```suggestion
training_roles: [author]
```

File: _profiles/james_vincent_mead.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+training_years: []              # in which years did you help out? (e.g. [2020, 2019])
```
Comment (klieret) [inline _profiles/james_vincent_mead.md:19]:
```suggestion
training_years: [2023]
```

**Review by klieret — CHANGES_REQUESTED** [review]

**Comment by klieret** [comment]
Thanks! Once you accept the suggestions, I'll merge :)

**Pull request merged**